oap-server/server-core/src/test/resources/component-libraries.yml

```diff
@@ -198,6 +198,12 @@ spring-cloud-gateway:
 RESTEasy:
   id: 62
   languages: Java
+SolrJ:
+  id: 63
+  languages: Java
+Solr:
```

Comment:
Is this what you will have in the other PR #2853 ? If so, leave it to where it should be

Reply:
This is for both server plugin and server mapping. So it makes sense to be here from I known.